.tx/config

```diff
@@ -0,0 +1,8 @@
+[main]
+host = https://www.transifex.com
+
+[eos-panel-extension.all]
+source_file = locale/eos-panel-extension.pot
+source_lang = en
+file_filter = locale/<lang>.po
```

Comment:
```suggestion
file_filter = po/<lang>.po
```

Reply:
I think you copied these from eos-hack-extension which uses the (unconventional) `locale/` directory.

Reply:
Same.